### DISCUSSION TITLE
Serve a deb repository instead of one-off deb packages

### DISCUSSION THREAD

**Discussion by GregJohnStewart** [Motion-Project/motion#1744] [General]
Hello, I plan on deploying a camera and don't want to have to come back to it often, relying on unattended upgrades to keep things up to date and secure.

However, it seems that Motion doesn't have much of an upgrade path, other than just installing another deb package, and to do this in an automated way is tedious.

Serving a debian repo is actually really, really easy ( https://earthly.dev/blog/creating-and-hosting-your-own-deb-packages-and-apt-repo/ ). Since you already have the packages served, it is only a couple more small steps to make it so `apt` can use it, and make deployments and upgrades much more fluid.

I can assist in this, as I have some repos served from github pages as well.

**Comment by GregJohnStewart** [comment]
Actually, it looks like Motion is in the Ubuntu repos already? The version I see from `apt-cache` is `4.3.1` though... That would make this OBE then, at least partially

**Comment by tosiara** [comment]
Thanks for the idea!
